### DISCUSSION TITLE
China discussion

### DISCUSSION THREAD

**Discussion by LuigiP57** [PygmalionOfCyprus/cmo-db-requests#4264] [General]
Inspired by @claudejdev, perhaps we can share information and suggestions about PLA here.

"All warfare is based on deception." - Sun Tzu.

**Comment by claudejdev** [comment]
Good idea, @LuigiP57 

Here are some English language resources I have (and that I'm happy to provide a scan, in fair use).

- Bond, Larry, and Carlson, Chris, and Grining, Peter, China’s Navy: Ships and Aircraft of the People’s Republic of China 1955 – 2021, 2021(+err. 2023)
- Bussert, James, and Elleman, Bruce, People's Liberation Army Navy Combat Systems Technology 1949-2010, 2011
- Friedman, Norman, Naval Institute Guide to World Naval Weapon System Fifth Edition, 2006
- Gardiner, Robert, Conway's All the World's Fighting Ships 1947-1995, 1995
- Gettinger, Dan, Unmanned Combat Aerial Vehicles, 2021
- Jane’s Aero Engines 2015-2016
- Jane’s All the World's Aircraft Production, Service, Unmanned 2021-2022
- Jane’s C4ISR Air/Joint/Land/Maritime2015/2021/2015[/i]
- Jane’s Fighting Ships 2021-2022
- Jane’s Land Warfare - Artillery & Air Defence/Armoured Fighting Vehicles/Logistical 2017-2018
- Jane’s Unmanned Maritime Vehicle 2021-2022
- Jane’s Weapons - Air/Infantry/Naval/Strategic 2021-2022
- Kerdiles, Jean-Luc, SA321 Super Frelon, 2012
- Meyer, Manfred, Modern China’s Maritime Forces Second Edition, 2024
- Prézelin, Bernard, Flottes de combat 2006
- Rupprecht, Andreas. Chinese Air Power in the 20th Century – Rise of the Red Dragon, 2019
- Rupprecht, Andreas. Dragon’s Wings, Chinese Fighter & Bomber Aircraft Development, 2013
- Rupprecht, Andreas. Modern Chinese Warplanes, Chinese Army Aviation – Aircraft and Units, 2018
- Rupprecht, Andreas. Modern Chinese Warplanes, Chinese Air Force – Aircraft and Units, 2018
- Rupprecht, Andreas. Modern Chinese Warplanes, Chinese Naval Aviation – Aircraft and Units, 2018
- Rupprecht, Andreas. Red Dragon 'Flanker', 2022
- Rupprecht, Andreas, and Cooper, Tom, Modern Chinese Warplanes, 2012
- Worldwide Equipment Guide 2016


In Chinese language, I have several issues of these magazines (I can't read the language, though):

- Naval and Merchant Ships 舰船知识
- Ordnance Knowledge 兵器知识
- Ordnance Industry Science Technology 兵工科技
- Ship Knowledge 舰船知识
- Shipborne Weapons 舰载武器

I can be found on the forum and on Discord (same name)

**Comment by LuigiP57** [comment]
I'd like to hear your thoughts on the rumored H-20 bomber and JH-XX/JH-26/H-18. fighter-bomber. Should they be added to DB under the Hypothetical category?
https://en.wikipedia.org/wiki/Xi%27an_H-20
![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/107970049/ca57fe6f-1afb-4c61-8f67-2266ca451865)
https://www.twz.com/25989/intel-report-confirms-china-developing-stealthy-regional-bomber-in-addition-to-strategic-bomber
![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/107970049/2642e578-fd70-4b50-a643-f4f4e06692c8)

**Reply by LuigiP57** [reply]
@claudejdev @HunterDNiu @tbrandon517 @CV60LCC19 @bostonmyk

**Comment by claudejdev** [comment]
https://www.sinodefenceforum.com/t/h-20-bomber-with-h-x-jh-xx.6451/

Here's a 10-year-long thread on Sinodefence forum about that project.

On one hand, it's a pipedream and, from what I understand, doesn't match current Chinese needs.

On the other hand, wilder concepts from the Cold War era are already in the database, so why not.

In my view, it would be a matter of time vs usefulness: will it be used in scenarios?

**Reply by LuigiP57** [reply]
Some kind of H-6 / JH-7 hybrid? Loadouts range from CJ-100 and YJ-21 to LRAAM and Chinese SDBs.

In fact, the H-6s in PLAAF is now more of a long-range strike aircraft, and the JH-7 is being replaced by the J-16. Perhaps we can think of it as a next-generation long-range fighter-bomber or the dawn of the sixth-generation aircraft?

Compared to the B-21, it has more flexiblity and air combat capability. The H-20 is more focused on strategic nuclear deterrence.

I bet someone will use it to replace the H-6s and JH-7s, or even the J-16.

**Comment by HunterDNiu** [comment]
> Some kind of H-6 / JH-7 hybrid? Loadouts range from CJ-100 and YJ-21 to LRAAM and Chinese SDBs.
> 
> In fact, the H-6s in PLAAF is now more of a long-range strike aircraft, and the JH-7 is being replaced by the J-16. Perhaps we can think of it as a next-generation long-range fighter-bomber or the dawn of the sixth-generation aircraft?
> 
> Compared to the B-21, it has more flexiblity and air combat capability. The H-20 is more focused on strategic nuclear deterrence.
> 
> I bet someone will use it to replace the H-6s and JH-7s, or even the J-16.

I've seen indications of this from 2022 in both CHinese and expo situations.

**Comment by LuigiP57** [comment]
Maybe this isn't the place to talk about these things, it's more convenient for us to talk directly under ticket. We have other places to have this kind of discussions, don't we?

**Comment by LuigiP57** [comment]
Close as no need
